lib/vimgolf/challenge.rb

```diff
@@ -11,7 +11,28 @@ def initialize(id)
       @id = id
     end
 
+    def local(infile, outfile)
+      @remote = false
```

Comment:
It's only used in the menu in play(), so local challenges don't show an option to upload the solution.